Sample-01/src/views/ExternalApi.js

```diff
@@ -114,13 +114,65 @@ export const ExternalApiComponent = () => {
         )}
 
         <h1>External API</h1>
+        <p class="lead">Ping an external API by clicking the button below.</p>
+
         <p>
-          Ping an external API by clicking the button below. This will call the
-          external API using an access token, and the API will validate it using
-          the API's audience value.
+          This will call a local API on port 3001 that would have been started
+          if you run <code>npm run dev</code>, using an access token. The API
+          will validate it using the API's audience value.
```

Comment:
I find the placing of the `, using an access token` read a bit weird.

```suggestion
          This will call a local API on port 3001 that would have been started
          if you run <code>npm run dev</code>. An access token is sent as part of the request's `Authorization` header and the API
          will validate it using the API's audience value.
```

Reply:
Good suggestion - thanks!